test/unit/mock/fake_runtime_client.py

```diff
@@ -442,7 +434,8 @@ def _get_job(self, job_id: str, exclude_params: bool = None) -> Any:
 
     # pylint: disable=unused-argument
     def list_backends(
-        self, hgp: Optional[str] = None, channel_strategy: Optional[str] = None
+        self,
+        hgp: Optional[str] = None,
```

Comment:
The `pylint disable` is no longer needed. Also, could the function be defined in a single line?